S/SLICOT/build_tarballs.jl

```diff
@@ -96,21 +105,6 @@ install_license ../LICENSE
 # platforms are passed in on the command line
 platforms = expand_gfortran_versions(supported_platforms())
 
-# The following was derived from #4770 (Pfapack)
-# https://github.com/JuliaPackaging/Yggdrasil/pull/4770
-# Since we need to link to libblastrampoline which has seen multiple
-# ABI-incompatible versions, we need to expand the julia versions we target
-julia_versions = [v"1.7", v"1.8", v"1.9", v"1.10"]
-function set_julia_version(platforms::Vector{Platform}, julia_version::VersionNumber)
-    _platforms = deepcopy(platforms)
-    for p in _platforms
-        p["julia_version"] = string(julia_version)
-    end
-    return _platforms
-end
-expand_julia_versions(platforms::Vector{Platform}, julia_versions::Vector{VersionNumber}) =
-    vcat(set_julia_version.(Ref(platforms), julia_versions)...)
-platforms = expand_julia_versions(platforms, julia_versions)
```

Comment:
Why was this removed?

Reply:
Oh, you're changing the required version of Julia.  But you can't do that without changing the version number of the package, otherwise you're breaking the previous build